bootloader/grub.go

```diff
@@ -300,3 +301,15 @@ func (g *grub) TryKernel() (snap.PlaceInfo, error) {
 	}
 	return nil, ErrNoTryKernelRef
 }
+
+// IsCurrentlyManaged returns true when the boot config is managed by snapd.
+//
+// Implements ManagedBootloader for the grub bootloader.
+func (g *grub) IsCurrentlyManaged() (bool, error) {
```

Comment:
do we need to distinguish between ubuntu-boot managed and ubuntu-seed managed in which case this should take a bootloader.Options too ?

Reply:
No, I don't think so.